frontend/src/components/pages/admin/posting/AdminPostingDetails.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import { VStack, Box, Container, Button, Flex } from "@chakra-ui/react";
+
+import { gql, useQuery } from "@apollo/client";
+import { useParams } from "react-router-dom";
+import { ChevronLeftIcon } from "@chakra-ui/icons";
+import { PostingResponseDTO } from "../../../../types/api/PostingTypes";
+import PostingDetails from "../../../common/PostingDetails";
+
+const POSTING = gql`
+  query Posting($id: ID!) {
```

Comment:
Small suggestion: let's prefix all operation names with the component they're located in, this is a common practice with GraphQL as it helps with debugging

```suggestion
  query AdminPostingDetails_Posting($id: ID!) {
```

Reply:
Done!